src/libs/actions/Policy.js

```diff
@@ -88,6 +90,15 @@ function getPolicyList() {
                     },
                 }), {});
                 Onyx.mergeCollection(ONYXKEYS.COLLECTION.POLICY, policyDataToStore);
+                const keysToClear = _.filter(_.keys(allPolicies), key => Str.startsWith(key, ONYXKEYS.COLLECTION.POLICY)
+                    && !_.keys(policyDataToStore).includes(key));
+                const keyValuePairsForClearing = _.map(keysToClear, key => [key, 'null']);
+                if (keyValuePairsForClearing.length > 0) {
+                    Onyx.multiSet(keyValuePairsForClearing);
+
+                    // Clear them in cache
+                    _.each(keysToClear, key => cache.set(key, null));
```

Comment:
I don't think we'll need to clear the Onyx cache manually. That's already happening [here](https://github.com/Expensify/react-native-onyx/blob/master/lib/Onyx.js#L515-L519).